docs/management/admin/third-party-actions.asciidoc

```diff
@@ -16,6 +16,8 @@ You can perform response actions on hosts enrolled in other third-party endpoint
 * Third-party response actions require an https://www.elastic.co/pricing[Enterprise subscription].
 
 * Each response action type has its own user role privilege requirements. Find an action's role requirements at <<response-actions>>.
+
+* Additional <<response-actions-config,configuration>> is required to connect {elastic-sec} with the third-party system.
```

Comment:
Suggesting a minor edit if this is a pre-req for connecting to _any_ third-party system. 

```suggestion
* Additional <<response-actions-config,configuration>> is required to connect {elastic-sec} with a third-party system.
```